Backend/Remora.Discord.API.Abstractions/API/Rest/IDiscordRestEmojiAPI.cs

```diff
@@ -56,8 +138,7 @@ Task<Result<IReadOnlyList<IEmoji>>> ListGuildEmojisAsync
     /// <param name="emojiID">The ID of the emoji.</param>
     /// <param name="ct">The cancellation token for this operation.</param>
     /// <returns>A retrieval result which may or may not have succeeded.</returns>
-    Task<Result<IEmoji>> GetGuildEmojiAsync
-    (
+    Task<Result<IEmoji>> GetGuildEmojiAsync(
```

Comment:
?

Reply:
Probably an error with auto-formatting?

Reply:
Yeah, I think my Rider did that automatically